README_ANSIBLE_CONTAINER.md

```diff
@@ -0,0 +1,29 @@
+# Running ansible in a docker container
```

Comment:
This file doesn't exist anymore, it was removed in commit 7fd854, the information about the container image is now in [README_CONTAINER_IMAGE.md](/openshift/openshift-ansible/blob/master/README_CONTAINER_IMAGE.md) and we should update that one instead as needed.